{{cookiecutter.project_slug}}/package.json

```
@@ -4,32 +4,34 @@
   "version": "{{cookiecutter.project_version}}",
   "description": "{{cookiecutter.project_description}}",
   "type": "module",
-  "main": "./src/main.js",
-  "exports": "./src/main.js",
+  "main": "./dist/main.js",
+  "exports": "./dist/main.js",
   "files": [
-    "src/**/*.js",
-    "!src/**/*.test.js"
+    "dist/**/*.js",
+    "dist/**/*.d.ts"
   ],
   "scripts": {
+    "build": "tsc",
     "prepare": "husky install node_modules/@netlify/eslint-config-node/.husky/",
     "prepublishOnly": "npm ci && npm test",
-    "test": "run-s format test:dev",
-    "format": "run-s format:check-fix:*",
-    "format:ci": "run-s format:check:*",
+    "prepack": "npm run build",
+    "test": "run-s build format test:dev",
+    "format": "run-s build format:check-fix:*",
+    "format:ci": "run-s build format:check:*",
     "format:check-fix:lint": "run-e format:check:lint format:fix:lint",
     "format:check:lint": "cross-env-shell eslint $npm_package_config_eslint",
     "format:fix:lint": "cross-env-shell eslint --fix $npm_package_config_eslint",
     "format:check-fix:prettier": "run-e format:check:prettier format:fix:prettier",
     "format:check:prettier": "cross-env-shell prettier --check $npm_package_config_prettier",
     "format:fix:prettier": "cross-env-shell prettier --write $npm_package_config_prettier",
-    "test:dev": "run-s test:dev:*",
-    "test:ci": "run-s test:ci:*",
+    "test:dev": "run-s build test:dev:*",
+    "test:ci": "run-s build test:ci:*",
     "test:dev:ava": "ava",
     "test:ci:ava": "nyc -r lcovonly -r text -r json ava"
   },
   "config": {
-    "eslint": "--ignore-path .gitignore --cache --format=codeframe --max-warnings=0 \"{src,scripts,.github}/**/*.{js,md,html}\" \"*.{js,md,html}\" \".*.{js,md,html}\"",
-    "prettier": "--ignore-path .gitignore --loglevel=warn \"{src,scripts,.github}/**/*.{js,md,yml,json,html}\" \"*.{js,yml,json,html}\" \".*.{js,yml,json,html}\" \"!**/package-lock.json\" \"!package-lock.json\""
+    "eslint": "--ignore-path .gitignore --cache --format=codeframe --max-warnings=0 \"{src,scripts,.github}/**/*.{js,ts,md,html}\" \"*.{js,ts,md,html}\" \".*.{js,ts,md,html}\"",
```

```suggestion
    "eslint": "--ignore-path .gitignore --cache --format=codeframe --max-warnings=0 \"{src,scripts,.github}/**/*.{js,cjs,mjs,ts,md,html}\" \"*.{js,cjs,mjs,ts,md,html}\" \".*.{js,cjs,mjs,ts,md,html}\"",
```

@ehmicky as we already have a common npm package for that is there a way to have that centralised?

---

Hum, good question. :thinking: 

The common package is for the ESLint configuration, which can be shared using the `extends` property. When it comes to CLI flags, we could only share by creating some binary in `@netlify/eslint-config-node` calling `eslint` with those flags, and then make our repository use that `eslint` wrapper.